rubric/rubric_presentations.md

```diff
@@ -11,22 +11,31 @@
     <th>Excellent (A- to A+)</th>
   </tr>
   <tr>
-    <th>Engagement & Clarity (40%)</th>
+    <th>Engagement & Clarity (30%)</th>
     <td>Not Attempted</td>
     <td>Speaker gives a presentation that is so short, shallow, or confusing that it was clear that no effort was made.</td>
     <td>Words are spoken quietly or the speaker does not enunciate so that few or no listeners can understand; speaker lacks enthusiasm; speaker pauses frequently and cannot remember what to say; speaker ignores the audience and never makes eye contact; speaker constantly reads off the slides.</td>
     <td>Speaker can make it through the presentation without significant stumbling; speaker mostly projects so that the evaluator can hear clearly; speaker occasionally makes eye contact with the audience.</td>
-    <td>Speaker projects so that everyone can hear almost everything, although there may be a couple moments where the speaker trails off; speaker enunciates clearly with relatively few umms and ahhs; eye contact is fairly consistent, and body language is mostly natural and open.</td>
-    <td>Speaker projects so that everyone can hear throughout the whole presentation; speaker enunciates clearly and enthusiastically without mumbling or slurring; eye contact is held and body language is used to enhance the verbal presentation; the speaker can smile and entertain the audience; the speaker does not simply read off the slides; the time limit is respected; the speaker looks to the audience or even interacts with them in order to gauge their understanding and adjust the pace accordingly.</td>
+    <td>Speaker projects so that everyone can hear almost everything, although there may be a couple moments where the speaker trails off; speaker enunciates clearly with relatively few umms and ahhs; eye contact is fairly consistent, and body language is mostly natural and open; audience engagement has been a consideration when designing the presentation, although it might not be perfectly executed.</td>
+    <td>Speaker projects so that everyone can hear throughout the whole presentation; speaker enunciates clearly and enthusiastically without mumbling or slurring; eye contact is held and body language is used to enhance the verbal presentation; the speaker can smile and entertain the audience; the speaker does not simply read off the slides; the time limit is respected; the speaker looks to the audience or even interacts with them in order to gauge their understanding and adjust the pace accordingly; audience engagement has been a priority in the presentation design, e.g. a captivating introduction or an interactive element during the presentation.</td>
```

Comment:
@mgelbart Since you were in charge of the 542 rubrics, what do you think of Varada's comment here? I personally agree that it is good to also cover the content of the presentations (and [some of this is covered by the lab3 rubrics in 542](https://github.ubc.ca/mds-2021-22/DSCI_542_comm-arg_instructors/blob/c40f176c9ef608cc8bbc21069d4c7caca09d6ab3/source/lab3/lab3.ipynb)), but I don't know if we leave this out of the presentation rubric intentionally because we want to use it in combination with the reasoning rubric or similar for assessing content?

Reply:
Added, thanks @kvarada !